be/src/olap/olap_common.h

```diff
@@ -199,6 +199,13 @@ struct Version {
     bool operator==(const Version& rhs) const {
         return first == rhs.first && second == rhs.second;
     }
+
+    // compared by start version, than end version
+    bool operator< (const Version& other) const {
+        if (first < other.first) return true;
```

Comment:
Now will appears (2,3), (1,4) version. So the less comparator will be ambiguous.

Reply:
OK, I will create a comparator for my own case.